src/EmailList.js

```diff
@@ -19,17 +20,19 @@ EmailList.defaultProps = {
 };
 
 const allUsersQuery = gql`
-  query users {
-    allUsers(orderBy: email_ASC) {
+  query users($date: DateTime) {
+    allUsers(orderBy: email_ASC, filter: {createdAt_gt: $date}) {
       id,
       email
-    }
-    
-  }
-`;
+    } 
+  }`;
 
 const EmailListWithData = graphql(allUsersQuery, {
-  options: () => ({}),
+  options: {
+    variables: {
+      date: new Date(format(subDays(new Date(), 7), 'YYYY-MM-DDTHH:mm:ss.SSSZ'))
```

Comment:
Is it necessary to wrap the `format` with a `new Date`? That seems odd...

Reply:
Right, this is because graphql is expecting a date type of DateTime. format() will spit out the ISO string so I wrapped it in new Date() to translate that as a DateTime type. I can double check if this is necessary later today!

Reply:
@wdoug good catch, this works with just the ISO string of the date. I'll get rid of this new Date().